### PR TITLE
Creates agentless CSPM troubleshooting page

### LINKED ISSUE #6044: [Request] Add agentless troubleshooting steps to Agentless doc
### Description
We need to create a new section for Agentless integration troubleshooting/FAQ. The Q/A we have come up with so far are:

When you make a new integration, how long until the agent appears on the CSPM integration policies page?
- The integration on the integration policies but it will have an “Add agent” button for up to a couple minutes. No action is needed, the agent enrolment is happening automatically but you may have to refresh the page to see the agent

Can I create an agentless integration if I changed my Fleet server host value?
- Fleet Server Host must be the default value
- [ ]  Need to find out how the user can tell whether the default value was changed, and what the error that appears would be

How can I delete the agentless integration?
- warn the customer that deletion will remove resources and stop data ingestion
- When we create agentless integration, new agent policy appears on the Fleet >Agent policies page — can’t use the “Delete integration” here. They should delete it from the CSPM Integration policies page

How do I troubleshoot an unhealthy or offline agent?
- If agent is unhealthy — refer to existing troubleshooting steps for unhealthy agent
- User would have to figure out which agent is associated with the agentless integration
- To identify which agent is related to the agentless integration, go to CSPM Integration page -> Integration policies page, then click agents for the "agentless" policies, then click the agent’s Host link > Logs tab

What if I enter the wrong credentials:
- If the user enters the wrong credentials agent will show as unhealthy -> User should then delete the agentless integration and set up again 
- [ ] Need to clarify - what is agent status in this case, and what would the logs show?

Romulo's related ticket: https://github.com/elastic/kibana/issues/198151

### Background & resources

* PRs: 
* Issues/metas: 
* Point of contact: 
* Test environments: 

### Which documentation set does this change impact?

ESS and serverless

### ESS release

8.16

### Serverless release

Dec 2

### Feature differences

-

### API docs impact

-

### Prerequisites, privileges, feature flags

_No response_

### PR DESCRIPTION
**Pull request by benironside**
Fixes #6044 — creates a troubleshooting page for the new agentless integration.

Preview: [Agentless integrations FAQ](https://security-docs_bk_6179.docs-preview.app.elstc.co/guide/en/security/master/agentless-integrations.html)

### CODE REVIEW

File: docs/getting-started/agentless-troubleshooting.asciidoc [hunk 1]
```diff
@@ -0,0 +1,42 @@
...
+[elastic_agent.cloudbeat][error] Failed to update registry: failed to get AWS accounts: operation error Organizations: ListAccounts, get identity: get credentials: failed to refresh cached credentials, operation error STS: AssumeRole, https response error StatusCode: 403, RequestID: cb01df90-1e78-4ae0-bdad-7e71b1d68f86, api error AccessDenied: User: arn:aws:iam::704479110758:user/lola is not authorized to perform: sts:AssumeRole on resource: arn:aws:iam::704479110758:role/cloudbeat-root
+```
++
+. For information about collecting logs, refer to {fleet-guide}/fleet-troubleshooting.html[Fleet troubleshooting]. 
+
+*How do I delete an agentless integration?*
+
+NOTE: Deleting your integration will remove all associated resources and stop data ingestion.
+
+When you create a new agentless CSPM integration, a new agent policy appears on the **Agent policies** tab of the **Fleet** page, but you can't use the **Delete integration** button on this page. Instead, delete delete the integration from the CSPM Integration policies page.
```
Comment (smriti0321) [inline docs/getting-started/agentless-troubleshooting.asciidoc:36]: Extra "Delete in this line- Instead, **delete** delete the integration from the CSPM Integration policies page.

File: docs/getting-started/agentless-troubleshooting.asciidoc [hunk 2]
```diff
@@ -0,0 +1,42 @@
...
+. For information about collecting logs, refer to {fleet-guide}/fleet-troubleshooting.html[Fleet troubleshooting]. 
+
+*How do I delete an agentless integration?*
+
+NOTE: Deleting your integration will remove all associated resources and stop data ingestion.
+
+When you create a new agentless CSPM integration, a new agent policy appears on the **Agent policies** tab of the **Fleet** page, but you can't use the **Delete integration** button on this page. Instead, delete delete the integration from the CSPM Integration policies page. 
+
+. Find **Integrations** in the navigation menu or use the {kibana-ref}/introduction.html#kibana-navigation-search[global search field], then search for and select `CSPM`.
+. On the CSPM integration page, go to the **Integation policies** tab.
```
Comment (smriti0321) [inline docs/getting-started/agentless-troubleshooting.asciidoc:39]: typo in **Integation policies**

**Review by smriti0321 — CHANGES_REQUESTED** [review]

File: docs/getting-started/agentless-troubleshooting.asciidoc [hunk 3]
```diff
@@ -0,0 +1,42 @@
+[[agentless-integration-troubleshooting]]
+= Agentless integrations FAQ
+
+Frequently asked questions and troubleshooting steps for {elastic-sec}'s agentless CSPM integration.
+
+*When I make a new integration, how long until the agent appears on the agent policies page?*
+
+After you create a new agentless integration, the new integration policy may show a button that says **Add agent** instead of the associated agent for several minutes during agent enrolment. No action is needed other than refreshing the page once enrolment is complete.
```
Comment (Omolola-Akinleye) [inline docs/getting-started/agentless-troubleshooting.asciidoc:8]: type: `enrolment` to `enrollment`

File: docs/getting-started/agentless-troubleshooting.asciidoc [hunk 4]
```diff
@@ -0,0 +1,42 @@
...
+
+*How do I troubleshoot an "Unhealthy" agent?*
+
+On the **Fleet** page, the agent associated with an agentless integration will have a name that begins with `agentless`. To troubleshoot an unhealthy agent:
+
+. Confirm that you entered the correct credentials for the cloud provider you're monitoring. The following is an example of an error log resulting from using incorrect AWS credentials:
++ 
+```
+[elastic_agent.cloudbeat][error] Failed to update registry: failed to get AWS accounts: operation error Organizations: ListAccounts, get identity: get credentials: failed to refresh cached credentials, operation error STS: AssumeRole, https response error StatusCode: 403, RequestID: cb01df90-1e78-4ae0-bdad-7e71b1d68f86, api error AccessDenied: User: arn:aws:iam::704479110758:user/lola is not authorized to perform: sts:AssumeRole on resource: arn:aws:iam::704479110758:role/cloudbeat-root
```
Comment (benironside) [inline docs/getting-started/agentless-troubleshooting.asciidoc:27]:
```suggestion

```

**Pull request closed without merging**